### ISSUE TITLE
Affection Reset to 0

### ISSUE THREAD

**Issue by Rai99** [Monika-After-Story/MonikaModDev#3641]
My affection was reset a couple days ago and I had to use a backup. I'm in unstable.  ~~Finally happens to me, rip~~

Persistent: Sent to @jmwall24 

[mas_log.txt](https://github.com/Monika-After-Story/MonikaModDev/files/2836522/mas_log.txt)

[aff_log.txt](https://github.com/Monika-After-Story/MonikaModDev/files/2836524/aff_log.txt)

**Comment by ThePotatoGuy** [comment]
these lines:
```
[2019-02-02 23:23:15.214000]: SAVE | 2274.0 | 2274.0 -> 2274.0
[2019-02-02 23:25:56.956000]: SAVE | 0 | 0 -> 0
```

Do you remember what you were doing dring these times? (11:23pm - 11:25pm)

**Comment by ThePotatoGuy** [comment]
in the meantime think we'll add another var as backup and reference it in `_mas_getAffection` so it doesn't ever return 0.

**Comment by Rai99** [comment]
I think that day, I was actually doing work on a different laptop, so I might have been doing absolutely nothing out of the ordinary. 

Actually I did have to do some crazy things, but that was all done and dealt with before that time. My steam couldn't connect to the servers so I tried all sorts of things like adding a -tcp argument in the shortcut, wiping and reinstalling everything except the exe and steamapps. I play MAS off of steam but launch it through steam as a non-steam game. That issue I had with steam was fixed at about 10:37.
